lib/bolt/puppetfile.rb

```diff
@@ -61,8 +61,13 @@ def self.parse(path, skip_unsupported_modules: false)
     #
     def write(path, moduledir = nil)
       File.open(path, 'w') do |file|
-        file.puts '# This Puppetfile is managed by Bolt. Do not edit.'
-        file.puts "moduledir '#{moduledir.basename}'" if moduledir
+        if moduledir
+          file.puts "# This Puppetfile is managed by Bolt. Do not edit."
+          file.puts "# For more information, see https://pup.pt/bolt-modules\n\n"
+          file.puts "# The following directive installs modules to the managed moduledir."
+          file.puts "moduledir '#{moduledir.basename}'\n\n"
```

Comment:
It might be clearer to have a couple extra empty `file.puts` calls below this line rather than append `\n\n`. Or this could be a heredoc. Or both!

Reply:
I'll do the extra `file.puts`. Anything to avoid heredocs. :P